src/marketplace/items/item-details.tsx

```diff
@@ -0,0 +1,228 @@
+import * as React from 'react';
+import injectSheet, { StyleSheet, ClassNameMap } from 'react-jss';
+
+import { Grid, Divider, FormGroup, FormControl } from '@material-ui/core'
+import { H2, H3 } from '../../typography/headings';
+import { P } from '../../typography/paragraph';
+import { TickIcon } from '../../icons/tick';
+import { UnlockIcon } from '../../icons/unlock';
+import { StyledButton } from '../../common/styled-button';
+
+const styles: StyleSheet = {
+  root: {
+    width: '946px',
+    height: '100%',
+    marginTop: '30px',
+    marginBottom: '30px',
+    border: 'solid 1px #303c49',
+    borderRadius: '4px'
+  },
+
+  title: {
+    margin: '20px'
+  },
+
+  icon: {
+    marginLeft: '20px'
+  },
+
+  header: {
+    backgroundColor: '#2a3540'
+  },
+
+  body: {
+    textAlign: 'justify',
+    margin: '20px',
+    color: '#fff',
+    fontFamily: 'ProximaNovaRegular',
+    fontSize: '16px',
+    fontWeight: 400,
+    lineHeight: 1.67
+  },
+
+  formControl: {
+    marginRight: '100px'
+  },
+
+  divider: {
+    backgroundColor: '#475768'
+  },
+
+  dividerWrapper: {
+    width: '100%'
+  },
+
+  formGroup: {
+    '& span': {
+      fontSize: '14px'
+    },
+
+    '& span strong': {
+      color: '#93b0c1',
+      fontSize: '16px',
+      paddingRight: '20px'
+    }
+  },
+
+  bullet: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+    width: '30px',
+    height: '28px',
+    borderRadius: '18px',
+    backgroundColor: '#3b4b59',
+    border: 'solid 1px #495b70',
+  },
+
+  requirement: {
+    paddingLeft: '20px',
+    color: '#93b0c1'
+  },
+
+  requirements: {
+    width: '400px',
+  },
+
+  description: {
+    marginTop: '0px'
+  }
+};
+
+export type ItemProps = {
+  name: string,
+  logo: string,
+  status: string,
+  integration: string,
+  description: string,
+  location: string,
+  year_launched: number,
+  coin_pairs: string,
+  maker_fee: string,
+  taker_fee: string,
+  fiat_payments: string,
+  fiat_supported: Array<string>,
+  margin_trading: string,
+  kyc_aml: string,
```

Comment:
shouldn't this be camel case?

Reply:
Oh, yeah we can do that it is because comes like this from the airtable. But as we can map to props we can also change to camel case.